gatsby-ssr.js

```diff
@@ -1,25 +1,10 @@
 const React = require('react');
 const fs = require('fs');
-const map = require('lodash/map');
-
-const { FORM_NAMES } = require('./src/contactFormConstants');
 
 const HEAD_CSS = fs.readFileSync('src/stylesheets/head-loaded-styles.css').toString();
 
-// <form netlify... /> is here to trigger the netlify bot into making form submissions available
-// for this website. If I put the tags on the form which actually performs the submission
-// then it doesn't work. I believe that form is rendered too dynamically to trigger the netlify
-// bot. This form doesn't do any actual work on the site for users.
-const embeddedForms = map(FORM_NAMES, (value) => (
-  <form name={value} netlify="true" netlify-honeypot="bot-field" hidden key="form">
-    <input type="email" name="email" />
-  </form>
-));
-
```

Comment:
I recently learned that none of this stuff is required when using Gatsby with Netlify. Removing it makes this PR simpler because I'm sending params other than `email` to Netlify forms. If I don't remove all this stuff, there is an expectation that these other params would also be added to this static form, in addition to the React form in `NetlifyFormCallToAction.js`